src/AccessibilityInsights.Extensions.GitHub/PlaceHolderTextBox.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using System;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Media;
+
+namespace AccessibilityInsights.Extensions.GitHub
+{
+    public class PlaceHolderTextBox : TextBox
```

Comment:
how would it be used in UI code?

Reply:
it is already used in the ConfigurationModelControl, the text input for the URL